README.md

```diff
@@ -95,19 +97,21 @@ For ClamAV, macOS users can do the following:
 - `source ~/.bash_profile`
 - `freshclam` (installs virus definitions)
 
-Both the front-end (`/web-client`) and API (`/web-api`) share code that exists in `/shared`. Before you can run either, you need to run `npm install` inside the `shared` directory.
+Both the front-end (`/web-client`) and API (`/web-api`) share code that exists in `/shared`. Before you can run either, you need to run `npm install` inside the top-level directory.
 
-- `cd shared && npm i`
```

Comment:
The top-level menu is inaccessible to keyboard navigation in Firefox or Safari, but works fine in Chrome. Please ensure that keyboard-based navigation permits access to all menu items.

Reply:
This is addressed, but it's not really an application issue as much as it is system / browser preference. See previous comment regarding Judge's Dashboard.